### PR TITLE
Arm: Correctly handle multi-reg COPY

### LINKED ISSUE #19448: [Ubuntu/arm32] JitStressRegs=8 CoreFX test failures
https://ci.dot.net/job/dotnet_coreclr/job/master/view/arm/job/jitstress/job/arm_cross_checked_ubuntu_corefx_jitstressregs8_tst/17/consoleFull

Many, or every, test fails with:
```
21:08:53 Unhandled Exception: System.NotSupportedException: Stream does not support seeking.
21:08:53    at System.IO.FileStream.get_Length()
21:08:53    at Internal.IO.File.ReadAllBytes(String path)
21:08:53    at System.TimeZoneInfo.TryLoadTzFile(String tzFilePath, Byte[]& rawData, String& id)
21:08:53    at System.TimeZoneInfo.TryGetLocalTzFile(Byte[]& rawData, String& id)
21:08:53    at System.TimeZoneInfo.GetLocalTimeZoneFromTzFile()
21:08:53    at System.TimeZoneInfo.GetLocalTimeZone(CachedData cachedData)
21:08:53    at System.TimeZoneInfo.CachedData.CreateLocal()
21:08:53    at System.DateTime.get_Now()
21:08:53    at Xunit.Sdk.TestAssemblyRunner`1.RunAsync()
21:08:53    at Xunit.Sdk.XunitTestFrameworkExecutor.RunTestCases(IEnumerable`1 testCases, IMessageSink executionMessageSink, ITestFrameworkExecutionOptions executionOptions)
21:08:53    at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__7_1(Object state)
21:08:53    at System.Threading.QueueUserWorkItemCallbackDefaultContext.<>c.<.cctor>b__5_0(Object state)
21:08:53    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
21:08:53 --- End of stack trace from previous location where exception was thrown ---
21:08:53    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
21:08:53    at System.Threading.QueueUserWorkItemCallbackDefaultContext.ExecuteWorkItem()
21:08:53    at System.Threading.ThreadPoolWorkQueue.Dispatch()
21:08:53    at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
21:08:53 ./RunTests.sh: line 128: 32432 Aborted                 $RUNTIME_PATH/dotnet xunit.console.netcore.exe XsltCompiler.Tests.dll -xml testResults.xml -notrait Benchmark=true -notrait category=nonnetcoreapptests -notrait category=nonlinuxtests -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing
```

Related to https://github.com/dotnet/coreclr/issues/19029?

Even if fixed, the job will probably fail with https://github.com/dotnet/coreclr/issues/19447.

@dotnet/arm32-contrib @CarolEidt

### PR DESCRIPTION
**Pull request by CarolEidt**
Fix #19448

### CODE REVIEW

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu x64 Formatting

**Comment by CarolEidt** [comment]
@dotnet/dnceng - any information about when ci.dot.net will be back online?

**Comment by mmitche** [comment]
@CarolEidt Looking at it now.

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu x64 Formatting
-

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu x64 Checked CoreFX Tests

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by mmitche** [comment]
Ubuntu x64 Formatting

**Comment by mmitche** [comment]
test Ubuntu x64 Formatting

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu x64 Checked CoreFX Tests

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu x64 Formatting

File: src/jit/codegenarmarch.cpp [hunk 1]
```diff
@@ -2059,56 +2059,30 @@ void CodeGen::genCodeForStoreOffset(instruction ins, emitAttr size, regNumber sr
...
 
 //------------------------------------------------------------------------
-// genRegCopy: Generate a register copy.
+// genRegCopy: Produce code for a GT_COPY node.
+//
+// Arguments:
+//    tree - the GT_COPY node
+//
+// Notes:
+//    This will copy the register(s) produced by this nodes source, to
```
Comment (BruceForstall) [inline src/jit/codegenarmarch.cpp]: nit: nodes => node's

File: src/jit/codegenarmarch.cpp [hunk 2]
```diff
@@ -2059,56 +2059,30 @@ void CodeGen::genCodeForStoreOffset(instruction ins, emitAttr size, regNumber sr
...
+//    the register(s) allocated to this GT_COPY node.
+//    It has some special handling for these casess:
```
Comment (BruceForstall) [inline src/jit/codegenarmarch.cpp]: nit: casess => cases

**Comment by BruceForstall** [comment]
@dotnet/jit-contrib

**Comment by CarolEidt** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test
@dotnet-bot test Ubuntu arm Cross Checked Innerloop Build and Test
@dotnet-bot test Windows_NT arm Cross Checked Innerloop Build and Test
@dotnet-bot test Windows_NT arm Cross Debug Innerloop Build
@dotnet-bot test Windows_NT arm64 Cross Checked Innerloop Build and Test
@dotnet-bot test Windows_NT arm64 Cross Debug Innerloop Build

**Comment by CarolEidt** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test
@dotnet-bot test Ubuntu arm Cross Checked Innerloop Build and Test
@dotnet-bot test Windows_NT arm Cross Checked Innerloop Build and Test
@dotnet-bot test Windows_NT arm Cross Debug Innerloop Build
@dotnet-bot test Windows_NT arm64 Cross Checked Innerloop Build and Test
@dotnet-bot test Windows_NT arm64 Cross Debug Innerloop Build

**Comment by CarolEidt** [comment]
@dotnet/dnceng - any suggestions for actually getting this tested? I've closed & reopened once, and I hate to do that again if it's just going to continue to fail on some percentage of the jobs.

**Comment by MattGal** [comment]
@CarolEidt unfortunately right now we're experiencing a serious issue with ci.dot.net where agents are getting lost and failing to connect, even after multiple reboots of the host.  Our Jenkins-iest @mmitche is on the case but neither of us know the actual root cause yet.  I won't stop you from trying but do note that anything which needs https://ci.dot.net/ is going to be adversely affected for a while.

**Comment by MattGal** [comment]
Tracking issue is https://github.com/dotnet/core-eng/issues/4164 .  My sincere apologies for the inconvenience.

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu arm Cross Checked Innerloop Build and Test
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu x64 Checked CoreFX Tests

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu x64 Checked Innerloop Build and Test

**Comment by CarolEidt** [comment]
@dotnet-bot test Ubuntu arm Cross Checked corefx_jitstressregs8 Build and Test

**Pull request merged**